dsl/webflux/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+logging.level.org.springframework.web: DEBUG
+logging.level.org.springframework.integration: DEBUG
```

Comment:
New line in the end of each file.
You can configure your IDE to do that for your on file save.